### ISSUE TITLE
[Bug]: Simple Goal Templates compound when overwritting

### ISSUE THREAD

**Issue by youngcw** [actualbudget/actual#1399]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue

### What happened?

It looks like the speed upgrade with the setZero change has made it so the simple template (`#template 10`) will compound when using the overwrite option instead of resetting to zero and then applying.  If I add an await to the setZero call it works as expected, but I imagine that loses most of the speed gains.  I haven't looked into if the math could be changed to overcome this.

### Where are you hosting Actual?

NAS

### What browsers are you seeing the problem on?

Chrome

### Operating System

Linux

**Comment by shall0pass** [comment]
Good catch!  The 'await setZero' is included in PR #1350.  It was missed in the earlier PR.  When I was testing it, everything was calculated as expected in all of the various scenarios I tried.   If #1350 is accepted, this will be resolved.

**Comment by kyrias** [comment]
Might make sense to extract that change to a separate PR that might hopefully get reviewed and merged a bit quicker.  :)

**Comment by youngcw** [comment]
> hopefully get reviewed and merged a bit quicker

or we could just pester Jed to merge 1350 :)
